birdhouse/components/scheduler/notebookdeploy_job.env

```diff
@@ -9,7 +9,7 @@ export BIRDHOUSE_SCHEDULER_NOTEBOOKDEPLOY_JOB="
     --rm --name notebookdeploy
     --volume /var/run/docker.sock:/var/run/docker.sock:ro
     --volume ${BIRDHOUSE_LOG_DIR}:${BIRDHOUSE_LOG_DIR}:rw
-    --volume ${COMPOSE_DIR}:${COMPOSE_DIR}:ro
+    --volume \${COMPOSE_DIR}:\${COMPOSE_DIR}:ro
```

Comment:
I was referring to all the `$` some are escaped and some are not, not just `$COMPOSE_DIR`.  I don't remember to what the difference between escaping or not.  What happen when you test those?

Reply:
Yes, this is intended.
Some variables are set when this file is sourced and others are set by environment variables passed to the container. This is the previous behaviour but made explicit here.

Is this the best way to do this? Probably not... but I'm trying to not break backwards compatibility too much. If you're OK with me breaking backwards compatibility even more I'm sure I can think of a better way to construct these YAML fields.

Reply:
You know what... I'm going to make this better/more inline with how the stack actually works now... hold on a bit I'm going to add some more changes

Reply:
FYI it's been a while I touched this code but maybe the all the env var passed to the container are probably for extra invocations by the entrypoint script.  It's probably not to keep the volume-mount directives as env var.

However, this is prior to the recent changes that all scripts will source `read-configs.include.sh` so maybe we need less of those env var.  Only a real test will confirm this.

Reply:
This is no longer relevant since [b923730](https://github.com/bird-house/birdhouse-deploy/pull/503/commits/b923730b6b8c1befd35677bab7946444c794e63a)